indico/modules/categories/templates/management/settings.html

```diff
@@ -0,0 +1,66 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_fieldset, form_footer %}
+
+{% extends 'categories/management/base.html' %}
+
+{% block title %}
+    {% trans %}Settings{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {{ form_header(form) }}
+        {{ form_rows(form, fields=form.BASIC_FIELDS) }}
+        {% call form_fieldset(_('Event Header')) %}
+            {{ form_rows(form, fields=form.EVENT_HEADER_FIELDS) }}
+        {% endcall %}
+    {% call form_footer(form) %}
+        <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}"
+               data-disabled-until-change>
+    {% endcall %}
+
+    {{ form_header(icon_form, classes='icon-form', action=url_for('.upload_icon', category)) }}
```

Comment:
`id="icon-form"`? Seems strange to use a class for this, you use it to select a specific form below